CMakeLists.txt

```diff
@@ -396,6 +396,11 @@ else()
   option(ENABLE_JEMALLOC "enable jemalloc" ON)
 endif()
 
+if(CMAKE_SYSTEM_NAME MATCHES "Windows")
+  message(STATUS "detected Windows; disabled jemalloc.")
+  option(ENABLE_JEMALLOC "enable jemalloc" OFF)
+endif()
```

Comment:
Why is this needed? `BuildJeMalloc.cmake` already skips itself if WIN32 is detected.